tests/kafkatest/services/console_consumer.py

```diff
@@ -285,7 +290,8 @@ def clean_node(self, node):
         if self.alive(node):
             self.logger.warn("%s %s was still alive at cleanup time. Killing forcefully..." %
                              (self.__class__.__name__, node.account))
-        JmxMixin.clean_node(self, node)
+        idx = self.idx(node)
+        self.jmx_tool.clean_node(idx, node)
```

Comment:
Need to check if `self.jmx_tool` is None here?

Reply:
good catch! thanks

Reply:
Actually there are a few places like this. Maybe we can figure out how to let jmx_tool always be set even if we're not scraping any JMX objects? I think that's how the mixin worked before.

Reply:
ya, that makes sense. the less special-casing the better